### PR TITLE
Pipeline docker setup

### PR DESCRIPTION
**Pull request by JosiahLiberatore**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# https://medium.com/@wkrzywiec/build-and-run-angular-application-in-a-docker-container-b65dbbc50be8
+# Stage 1:  Build 
+
+FROM node:12.7-alpine AS build
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
+
+RUN npm install
```
Comment (escalonn) [inline Dockerfile]: let's copy package-lock.json also, and then run `npm ci` instead.

**Review by escalonn — CHANGES_REQUESTED** [review]
massive diff to package-lock should have been noticed. in this case it's due to different people using different versions of node (and specifically npm), the way to resolve this with the least friction (diffs like this back and forth) is for everyone to use the same version.

there is a spurious package-lock.json added to the root directory also, this came from running npm i in the wrong place. should notice this when committing or pushing. there's also an extra nginx conf. if the build context is `fakebook` then remove the one in the root dir.

File: Dockerfile (hunk 1 above)
Comment (dannydjg16) [reply Dockerfile]: We added these two commands and figured out that our problem was we placed the Dockerfile and other files in the wrong place (they were up one directory too high). So we think that the other comment you had has also been addressed correctly. As of right now, I was able to build and run the Dockerized app locally so I think this is ready to merge. The last thing we were unsure of was the diff to package-lock but as long as everyone uses the same Dockerfile(which means using the same node version), it should be correct I believe.

Comment (escalonn) [reply Dockerfile]: @dannydjg16 if it's ready for re-review then mark this conversation as resolved and re-request review from me in the top right

Comment (escalonn) [reply Dockerfile]: it's not ready though cause there's a huge diff to package-lock which has nothing to do with the purpose of this PR.
if we're going to standardize on a version of npm that uses lockfileversion 1, then i could let this through, but i'd better not see a massive diff changing the file back in another PR. get some other people to run `npm i` and see if they get lockfileversion 1 or 2 at the top of package-lock. whichever is more common, everyone else get on the version of node/npm that those people have.

Comment (escalonn) [reply Dockerfile]: cc @yetanothersegfault

Comment (yetanothersegfault) [reply Dockerfile]: We have removed the package-lock.json from this PR. I am going to ensure that we have consistency moving forwards by ensuring that everyone is using npm 7.x to make us all in line with the original scaffold

**Review by escalonn — APPROVED** [review]

**Pull request merged**